cabal.project

```diff
@@ -90,3 +87,20 @@ benchmarks: true
 
 -- The only sensible test display option
 test-show-details: streaming
+
+if impl(ghc < 9.8)
+  program-options
+    ghc-options: -Werror
+
+if impl(ghc >= 9.8)
+  -- This commit is on master of the plutus repo.
+  source-repository-package
```

Comment:
s-r-ps in conditionals don't work well, unfortunately.

I think you could just make this unconditional: it would be weird to have the ledger only sometimes depend on master of `plutus` for 9.8, in general that might not compile. Whereas making it always depend on master (for this branch which I assume you're not going to merge soon) is at least consistent.

Reply:
I made it unconditional.